### PR TITLE
Heartline Coaster Ratings Fixes

### PR DESCRIPTION
**Pull request by pizza2004**
I decided I'd try to create a pull request to match what Intel Orca wanted changed with #2119.  I partly just wanted to see how the whole pulling and such system worked with git, and hadn't done anything in coding for a while, so I thought I'd submit it just because.  Sorry if I did anything wrong or overstepped my bounds.

### CODE REVIEW

**Comment by pizza2004** [comment]
Yeah, definitely taking some time to get used to git.  I've used it before, but never really got very deep into it, mostly in projects that involved only me and one other person, where it didn't matter if I made a bunch of really small commits.  I had to do some stuff to revert commits, that's why this closed and then opened.

**Comment by janisozaur** [comment]
it's perfectly fine to do that, you just don't have to push it all. I sometimes work on more than one branch, where one is WIP version of the other and possibly there are more to test various things out.

Only after I've cleaned it up on feature-master branch, I do a push, taking care to provide some meaningful description in the commit messages etc.

**Comment by pizza2004** [comment]
Ah, yeah.  Makes sense.  I've never been certain how to use branching on git.  I had SVN pretty well figured out about... 6 years ago?  Yeah, something like that, but after Commander Genius switched over to Github I mostly just used it the same as I did before.

**Comment by IntelOrca** [comment]
Please can you not interleave the #ifdef and #else with a C if else statement, it makes it hard to follow.

**Comment by pizza2004** [comment]
Alright, I can make that change.  Hold on a moment.

**Comment by pizza2004** [comment]
Okay, does this look better?  I'm trying to make it as visually appealing as possible, but if it's no good I can make more changes.

**Comment by e-foley** [comment]
@pizza2004, I hope you don't mind, but I incorporated the proposed changes in [my original pull request](https://github.com/OpenRCT2/OpenRCT2/pull/2119).  Let me know if I missed anything.

**Comment by pizza2004** [comment]
I certainly don't mind either way.  I'd say it more depends on which IntelOrca thinks has the better formatting.  You did everything just the way he wanted it, I just didn't want to wait forever if you never came back, and figured it was something simple I could do on my own, just to try things out.

**Comment by pizza2004** [comment]
Oh, also, my PR addresses the nausea not being affected until 96 airtime is reached bug even with the original excitement rating implementation, which I thought was an important bug to fix whether someone believes that the excitement calculation should be changed or not.

**Comment by e-foley** [comment]
I chose not to incorporate the nausea modifications under `ORIGINAL_RATINGS` since doing so would make the flag name rather misleading, IMO.  Of course, there's room for debate.

**Comment by pizza2004** [comment]
Does it actually do the nausea thing in the original game?  I was under the impression it was more likely a bug in the implementation of OpenRCT2, but I wouldn't be that surprised either way.
